.config/waybar/style.css

```diff
@@ -86,3 +86,11 @@ window#waybar.solo {
 #bluetooth.connected {
     color: rgba(115, 186, 37, 1);
 }
+
+#scratchpad.empty {
+    background-color: transparent;
+}
+
+#scratchpad {
+    background: rgba(0, 0, 0, 0.2);
+}
```

Comment:
Why do you need to make it darker? This looks strange to me:
![image](https://github.com/user-attachments/assets/b87b992c-73e1-4ed0-bac8-88c042350009)

Can't it just have the same background color as the rest?

Reply:
oh, true :) I didn't notice it, copy lasted from the examples